dvclive/keras.py

```diff
@@ -1,4 +1,6 @@
-from tensorflow.keras.callbacks import Callback
+from tensorflow.keras.callbacks import (  # pylint: disable=no-name-in-module
```

Comment:
This seems like a weird change, but I'm guessing it's auto formatting? 🤷

Reply:
Seems like it started to occur recently, probably its some new rule for `pylint`. I need to research it, but don't want to block smaller changes.